modules/rtm/R/invert.auto.R

```diff
@@ -67,11 +67,30 @@ invert.auto <- function(observed, invert.options, return.samples=TRUE, save.samp
     i.try <- 1
     while(try.again & i.try <= n.tries){
         print(sprintf("Attempt %d of %d", i.try, n.tries))
-        samps.list <- list()
-        for(chain in 1:nchains){
-            print(sprintf("Chain %d of %d", chain, nchains))
-            invert.options$inits <- inits.function() 
-            samps.list[[chain]] <- invert.custom(observed=observed, invert.options=invert.options, quiet=quiet)
+        if(parallel){
+            if(!require(parallel)) stop("'parallel' package not installed")
```

Comment:
Thanks for the suggestions! I changed the behavior so that the function throws a warning and proceeds without parallelization if `parallel=TRUE` but the `parallel` package isn't installed.